packages/amplication-client/src/Application/ApplicationHome.tsx

```diff
@@ -20,8 +20,8 @@ import useNavigationTabs from "../Layout/UseNavigationTabs";
 import InnerTabLink from "../Layout/InnerTabLink";
 import { ApiTokenList } from "../Settings/ApiTokenList";
 import RouteWithAnalytics from "../Layout/RouteWithAnalytics";
-import ApplicationDatabaseSettingsForms from "./ApplicationDatabaseSettingsForms";
-import ApplicationAuthSettingForm from "./ApplicationAuthSettingForm";
+import ApplicationDatabaseSettingsForms from "./forms/db/ApplicationDatabaseSettingsForms";
```

Comment:
the folder name "form" is too generic. Also, for the context of this fix - please leave the folder structure as it is and leave the form components in the Application folder.